cmd/up/up.go

```diff
@@ -253,19 +253,21 @@ func Up(at analyticsTrackerInterface, ioCtrl *io.IOController, k8sLogger *io.K8s
 			}
 
 			up := &upContext{
-				Manifest:          oktetoManifest,
-				Dev:               nil,
-				Exit:              make(chan error, 1),
-				resetSyncthing:    upOptions.Reset,
-				StartTime:         time.Now(),
-				Registry:          registry.NewOktetoRegistry(okteto.Config{}),
-				Options:           upOptions,
-				Fs:                afero.NewOsFs(),
-				analyticsTracker:  at,
-				analyticsMeta:     upMeta,
-				K8sClientProvider: okteto.NewK8sClientProviderWithLogger(k8sLogger),
-				tokenUpdater:      newTokenUpdaterController(),
-				builder:           buildv2.NewBuilderFromScratch(at, ioCtrl),
+				Manifest:                     oktetoManifest,
+				Dev:                          nil,
+				Exit:                         make(chan error, 1),
+				resetSyncthing:               upOptions.Reset,
+				StartTime:                    time.Now(),
+				Registry:                     registry.NewOktetoRegistry(okteto.Config{}),
+				Options:                      upOptions,
+				Fs:                           afero.NewOsFs(),
+				analyticsTracker:             at,
+				analyticsMeta:                upMeta,
+				K8sClientProvider:            okteto.NewK8sClientProviderWithLogger(k8sLogger),
+				tokenUpdater:                 newTokenUpdaterController(),
+				builder:                      buildv2.NewBuilderFromScratch(at, ioCtrl),
+				unhandledTransientMaxRetries: 100,
```

Comment:
this `100` is very arbitrary, I think that `100` could be a good value to start testing with because we need a value that is a bit high, in case the retries need to run for hours overnight.

Reply:
How are we going to know if it's high or low? do we have any metric that we could add in order to know what's the maximum value?

Reply:
@jLopezbarb with the new debug log added with c488d90b, we should be able to see the total max retries and current count, when looking at the logs. Do you have more ideas?